README.md

```diff
@@ -54,6 +62,32 @@ When the install is complete, go ahead and run this in your terminal:
 lounge --help
 ```
 
+## Configure
+
+After installing and first run, lounge generates config file to your home directory and uses it. If you want to use other configuration or run multiple instances of Lounge, you need to use different home directory for Lounge.
+
+```
+lounge --home ~/test #uses test directory as home
+```
```

Comment:
I think the website says everything, but I guess we could add it here. Thoughts, @thelounge/maintainers?

If this paragraph stays in the PR, we'll work on improving it prior to merging :-)

Note that I really don't like that running `lounge` generates a config file (even when running `lounge --help`!!). I think it would be more suitable to create that file as part of the installation step, and/or `lounge init` maybe. Though this is a different discussion...